src/app/portlets/rule-engine/custom-types/visitors-location/visitors-location.component.ts

```diff
@@ -20,20 +20,27 @@ interface VisitorsLocationParams {
 const UNITS = {
```

Comment:
You can disable the whole file if you comment on top of the file instead of each line.